server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
```

Comment:
See the comment in `shared_case.py` about importing classes from modules.

Organizing imports, too.

Reply:
fixed